tcms/xmlrpc/serializer.py

```diff
@@ -151,6 +196,7 @@ class QuerySetBasedXMLRPCSerializer(XMLRPCSerializer):
     extra_fields = {}
 
     def __init__(self, model_class, queryset):
+        super().__init__(model_class, queryset)
```

Comment:
This looks a bit suspicious. Need to figure out is it really needed or not and maybe add some tests here. Leaving for further review for now.